odoo_install.sh

```diff
@@ -29,14 +29,17 @@ OE_PORT="8069"
 OE_VERSION="11.0"
 # Set this to True if you want to install Odoo 11 Enterprise!
 IS_ENTERPRISE="False"
+# Set this to True if you want to install Nginx!
+INSTALL_NGINX="True"
```

Comment:
Perhaps it is safer to default to `False`?

Reply:
I kind of always install it along with nginx hence it was set to true